source/containers/auth/Settings/views/AboutView/index.tsx

```diff
@@ -14,10 +15,12 @@ import styles from './index.scss';
 // }
 
 const AboutView: FC = () => {
+  const [ version ] = useState(browser.runtime.getManifest().version)
+
   return (
     <div className={styles.about}>
       <span>Stargazer Wallet Chrome Extension v2.1</span>
```

Comment:
Missed version on this line.

Reply:
using format: major.minor.patch, on this line we show major.minor . i.e. v2.1

Reply:
Fixed